actioncable/lib/rails/generators/channel/templates/assets/cable.js

```diff
@@ -0,0 +1,13 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
+// You can generate new channels where WebSocket features live using the rails generate channel command.
+//
+//= require action_cable
+//= require_self
+//= require_tree ./channels
+
+(function() {
+  this.App || (this.App = {});
+
+  App.cable = ActionCable.createConsumer();
+
+}).call(this);
```

Comment:
We already have a `cable.js` in application templates in railties. Should we refer to it instead of duplicating here? I have seen similar duplicate file for application_mailer.rb too in mailer_generator template. We can remove duplicate file there too.